pontoon/contributors/static/css/contributor.css

```diff
@@ -30,60 +30,37 @@ a.avatar:hover .desc ~ img {
   opacity: 0.3;
 }
 
-#username {
-  color: #EBEBEB;
-  font-size: 48px;
-  letter-spacing: -2px;
-  text-transform: none;
-}
-
-#username .uneditable {
-  letter-spacing: normal;
-  padding: 8px 0 20px;
-}
-
-#username .container {
-  display: inline-block;
+#profile-form {
+  display: block;
   position: relative;
+  text-align: left;
+  margin: 0 auto;
+  width: 620px;
+}
+#profile-form .field {
+  margin-top: 10px;
 }
 
-#username .container input {
-  background: transparent;
-  border-bottom: 1px dashed transparent;
+#profile-form .field input {
   color: #FFFFFF;
-  font-size: 48px;
-  font-weight: 300;
-  padding: 4px 50px;
-  text-align: center;
-  width: 100%;
+  background: #373C42;
+  border: 1px solid #4F5965;
+  border-radius: 5px;
```

Comment:
We should use the exact same styling as in other input fields, e.g. in the Filter projects field:
https://pontoon.mozilla.org/projects/

That is:
```
background: #333941;
border: 1px solid #4d5967;
border-radius: 3px;
```